src/lib/services/access-service.ts

```diff
@@ -228,7 +247,7 @@ export class AccessService {
                 await this.store.removeRolesOfTypeForUser(
                     userId,
                     RoleType.ROOT,
-                );
+                ); // TODO
```

Comment:
TODO?

Reply:
Addressed in https://github.com/Unleash/unleash/pull/3975/commits/4fb9eb691ff6cfbd1e449eed7eb0ed68867247f2